### DISCUSSION TITLE
Server connection error after enabling collaborative editing

### DISCUSSION THREAD

**Discussion by danigamba** [outline/outline#3637] [Self Hosting] [answered]
After upgrading from my company docker-compose installation to 0.64.3 we tried to enable collaborative editing.
Now we cannot edit any document nor disable the collaborative editing feature (i saw on another thread that is intended this way).
Even if is ok for us to have collaborative editing, the editor is totally unresponsive with "Server connection lost" icon in the bottom.
Could anyone help us figure it out?
Tested on both chrome desktop and mobile

**Comment by danigamba** [comment]
Adding a few details
I've managed to resolve issues with certificates (unsecure connection) and the "server connection lost" adding the https-portal environment variables (1.2.3.4 is the public ip of the server).
```
DOMAINS=my.domain.com -> http://1.2.3.4:3000
WEBSOCKET=true
STAGE=production
```
Altough now I'm facing these issues
* if I open a new clear session in incognito I'm not being redirected to the Microsoft Login 
* with a past session in the browser where I'm already logged in I can edit only a couple of files, all the others are blocked

**Reply by tommoor** [reply]
The restriction to disable collaborative editing is only in the HTML btw, you can bypass it by just editing the disabled state of that input if you need to put it back in an "emergency" situation.

**Comment by danigamba** [comment, marked as answer]
I solved all the issues doing a new deployment with the copied folders under a different third level domain and re-creating the Azure app and secret.
I really don't know what went wrong, please let me know if I can provide additional details.
